octoprint_mrbeam/software_update_information.py

```diff
@@ -309,7 +309,9 @@ def _get_package_description(
     if "{tier}" in branch_default:
         branch_default = branch_default.format(tier=get_tier_by_id(tier))
     if prerelease_channel and "{tier}" in prerelease_channel:
-        kwargs.update(prerelease_channel=prerelease_channel.format(tier=get_tier_by_id(tier)))
+        kwargs.update(
```

Comment:
Only format ?, is this format needed ?

Reply:
no this was some black reformatting and ended up in a commit